Assets/Scripts/GameController.cs

```diff
@@ -13,7 +13,7 @@ public class GameController : MonoBehaviour {
     // To be set in inspector
     public int numAIs = 1;
 
-    public static bool hasGameStarted = false;
```

Comment:
is this gonna mess anything up

Reply:
sure isssssssnt

Reply:
so this is to say the ball is not bound by the race, if there is no race the ball can move, but if there is a race the ball is bound to wait for the timer to end.